jsonnet/kube-prometheus/components/external-mixins.libsonnet

```diff
@@ -0,0 +1,58 @@
+local defaults = {
+  namespace: error 'must provide namespace',
+  commonLabels:: {
+    'app.kubernetes.io/name': 'kube-prometheus',
+    'app.kubernetes.io/component': 'external-mixins',
+    'app.kubernetes.io/part-of': 'kube-prometheus',
+  },
+  ruleLabels: {
+    role: 'alert-rules',
+  },
+};
```

Comment:
What is the value of adding default values when there is no default component? This way we have mostly incorrect labels and in current form, this doesn't provide any guidance as to what is configurable.